style.css

```diff
@@ -124,6 +124,142 @@ header {
   padding-right: 5%;
 }
 
+.Recentwork1 {
+  z-index: 1;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  opacity: 0;
+  visibility: hidden;
+  position: fixed;
+  overflow: scroll;
+  left: 0;
+  background-color: #fff;
+  justify-content: center;
+  padding: 45px 10px 30px 10px;
+  transition: visibility 0s ease 0.5s;
+}
+
+.Recentwork1.active {
+  visibility: visible;
+  opacity: 1;
+}
```

Comment:
- [ ] Your popup window covers the entire page, not creating the feeling of a popup window, and this way the user can misunderstand and think that he/she was redirected to another page. Kindly add margins to the popup and add some background filter with some dark overlay so your popup looks like it is floating above the content of the page.
 -Your design:
Mobile:
<img width="492" alt="image" src="https://user-images.githubusercontent.com/61919617/196674153-f80f47e7-279e-4497-a04c-ba877ce514bb.png">
Desktop:
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/61919617/196674243-f0f1d082-dd82-4f5b-9037-7dd39681ddc5.png">


-Figma template:
Mobile:
<img width="290" alt="image" src="https://user-images.githubusercontent.com/61919617/196674330-1bf86889-1ce1-46e4-975d-126780167895.png">

Desktop:
<img width="1069" alt="image" src="https://user-images.githubusercontent.com/61919617/196674418-bf05a08d-f450-4232-bd6e-42ae8835c99d.png">